### ISSUE TITLE
punctuate: index out of range

### ISSUE THREAD

**Issue by mifisignal** [go-loremipsum/loremipsum#3]
```
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
gopkg.in/loremipsum%2ev1.(*LoremIpsum).punctuate(0x400026d290, {0x400047a000, 0x0, 0xba})
	/go/pkg/mod/gopkg.in/loremipsum.v1@v1.1.0/util.go:29 +0x24c
gopkg.in/loremipsum%2ev1.(*LoremIpsum).Sentence(0x400026d290)
	/go/pkg/mod/gopkg.in/loremipsum.v1@v1.1.0/loremipsum.go:51 +0x78
gopkg.in/loremipsum%2ev1.(*LoremIpsum).SentenceList(0x400026d290, 0xa)
	/go/pkg/mod/gopkg.in/loremipsum.v1@v1.1.0/loremipsum.go:59 +0x7c
main.main()
	/build/cmd/publish/main.go:34 +0x17c
```

**Comment by mifisignal** [comment]
Looks like it's possible for the `sentence` argument to be an empty list.

**Comment by dnagir** [comment]
It seems like this should had been closed by #4
